scripts/fish/README.md

```diff
@@ -1,4 +1,5 @@
 # fish completions
 
-To enable bazel completions, copy //scripts/fish/completions/bazel.fish to
-~/.config/fish/completions/bazel.fish.
+To enable bazel completions, run `bazel build //scripts:fish_completion` and
```

Comment:
In a separate PR, it would be useful to add that info to https://github.com/bazelbuild/bazel/blob/master/scripts/packages/template_bin.sh#L181 as well.

Reply:
See https://github.com/bazelbuild/bazel/pull/12267